components/brave_wallet/browser/keyring_service.cc

```diff
@@ -140,17 +140,18 @@ const int kDiscoveryAttempts = 20;
 const char kKeyringNotFound[] = "";
 
 std::string GetRootPath(const std::string& keyring_id) {
-  std::string root(kRootPath);
-  auto coin = GetCoinForKeyring(keyring_id);
-  auto coin_part = std::to_string(static_cast<int32_t>(coin));
-  if (keyring_id == brave_wallet::mojom::kFilecoinTestnetKeyringId) {
-    coin_part = "1";
-  }
-  base::ReplaceSubstringsAfterOffset(&root, 0, "{coin}", coin_part);
-  if (coin == mojom::CoinType::ETH || coin == mojom::CoinType::FIL) {
-    base::StrAppend(&root, {"/0'/0"});
+  if (keyring_id == mojom::kDefaultKeyringId) {
+    return "m/44'/60'/0'/0";
+  } else if (keyring_id == mojom::kSolanaKeyringId) {
+    return "m/44'/501'";
+  } else if (keyring_id == mojom::kFilecoinKeyringId) {
+    return "m/44'/461'/0'/0";
+  } else if (keyring_id == mojom::kFilecoinTestnetKeyringId) {
+    return "m/44'/1'/0'/0";
   }
-  return root;
+
+  NOTREACHED();
+  return "";
```

Comment:
That was very hard to track what are exact values of root paths and 'algorithmic' way does not add any value here. Hardcoded them.